<launch>
   <node name="SDI_cmd" pkg="software_driving_interface" type="SDI_cmd" />
   <node name="SDI_state"   pkg="software_driving_interface" type="SDI_state"   />
</launch>
